warehouse/observations/tasks.py

```diff
@@ -149,3 +154,127 @@ def report_observation_to_helpscout(task, request: Request, model_id: UUID) -> N
     model.additional["helpscout_conversation_url"] = new_convo_location
     request.db.add(model)
     return
+
+
+@tasks.task(
+    bind=True,
+    ignore_result=True,
+    acks_late=True,
+    autoretry_for=(RequestException,),
+    retry_backoff=True,
+)
+def auto_quarantine_project(task, request: Request, model_id: UUID) -> None:
```

Comment:
nit: I wonder if this could have a better name. When reading the call in `execute_observation_report` I got confused why it wasn't conditional. I wish I had an obvious answer that wasn't just `maybe_auto_quarantine_project`

Reply:
Agreed. `evaluate_project_quarantine`?

Reply:
Good points, thanks - I'll also rename execute_observation_Report since that's no longer what it does.
How about `evaluate_project_for_quarantine()`?